timeless/templates/items/list.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'base.html' %}
+
+{% block header %}
+    <h1>{% block title %}Items{% endblock %}</h1>
+{% endblock %}
+
+{% block content %}
+    {% for item in items %}
+        <article class="item">
+        <header>
+            <div>
+            <h1>{{ item['id'] }}</h1>
```

Comment:
@ammaratef45 Do we only display the item's ID?

Reply:
The templates should be continued as in the puzzle I left.